docs/with-js.md

```diff
@@ -0,0 +1,173 @@
+If you have not yet migrated your project to TypeScript or you are in the process of migrating your project, the Glint CLI and language server will still function properly.
+
+## Setup
+
+All the Glint setup steps are identical.
+
+{% hint style="info" %}
+
+We do mean *identical*: unlike most existing JavaScript projects, you will need to include TypeScript in your dev dependencies.
+
+{% endhint %}
+
+Thus, installation for a JS setup looks like this:
+
+{% tabs %}
+{% tab title="Yarn" %}
+
+```sh
+yarn add --dev typescript @glint/core @glint/environment-ember-loose
+```
+
+{% endtab %}
+{% tab title="npm" %}
+
+```sh
+npm install -D typescript @glint/core @glint/environment-ember-loose
+```
+
+{% endtab %}
+{% endtabs %}
+
+{% code title="tsconfig.json" %}
```

Comment:
One note: I think this either needs to be `jsconfig.json` _or_ you need to explicitly set `allowJs: true` in your `compilerOptions`.

Reply:
Doh, thank you.

```suggestion
{% code title="jsconfig.json" %}
```